src/coreclr/jit/emitarm64.h

```diff
@@ -340,6 +340,9 @@ static code_t insEncodeReg_V_21_to_17(regNumber reg);
 // Return an encoding for the specified 'R' register used in '21' thru '17' position.
 static code_t insEncodeReg_R_21_to_17(regNumber reg);
 
+// Return an encoding for the specified 'R' register used in '20' thru '16' position.
+static code_t insEncodeReg_R_20_to_16(regNumber reg);
```

Comment:
I have fixed this in #96082.

Reply:
Thank you for letting me know. It looks like your PR will get merged in pretty soon, so I'll wait to merge your changes into mine.

Reply:
@amanasifkhalid  - do you mind addressing https://github.com/dotnet/runtime/pull/96082#discussion_r1430217591 in your PR, since you will have another round of CI run anyway?

Reply:
Sure thing, I'll update the assert.